cypress/e2e/marc-authority/verify-authority-source-facet-option.cy.js

```diff
@@ -0,0 +1,67 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import MarcAuthorityBrowse from '../../support/fragments/marcAuthority/MarcAuthorityBrowse';
+
+const testData = {};
+const jobProfileToRun = 'Default - Create SRS MARC Authority';
+const fileName = 'marFileForC365630.mrc';
+const updatedFileName = `testMarcFileUpd.${getRandomPostfix()}.mrc`;
+const authoritySource = 'LC Subject Headings (LCSH)';
+let createdAuthorityID;
+
+describe('Importing MARC Authority files', () => {
```

Comment:
Please, change describe's name regarding tesrail folder
And move file to proper folder marc>marc authority> browse authority records